### ISSUE TITLE
Error in pip install

### ISSUE THREAD

**Issue by norecces** [napjon/krisk#70]
```pip install krisk
Collecting risk
Using cached krisk-0.3.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/c4/yxk5p2k93z53gw5ywq750x3nhlvp8p/T/pip-build-imEuXs/krisk/setup.py", line 2, in <module>
        from krisk import __version__
      File "krisk/__init__.py", line 5, in <module>
        import krisk.plot.api as plot
      File "krisk/plot/api.py", line 1, in <module>
        from krisk.plot.make_chart import make_chart
      File "krisk/plot/make_chart.py", line 2, in <module>
        from krisk.chart.api import Chart
      File "krisk/chart/api.py", line 1, in <module>
        from krisk.chart.core import rcParams, Chart
      File "krisk/chart/core.py", line 189
    SyntaxError: Non-ASCII character '\xef' in file krisk/chart/core.py on line 189, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details```

**Comment by napjon** [comment]
What Python version is this?

**Comment by norecces** [comment]
2.7 :)

**Comment by napjon** [comment]
Solved by #71
